packages/orchestration/src/facade.js

```diff
@@ -39,36 +43,41 @@ const makeLocalChainFacade = localchain => {
       return mockLocalChainInfo;
     },
 
-    // @ts-expect-error FIXME promise resolution through membrane
     async makeAccount() {
       const account = await E(localchain).makeAccount();
 
       return {
-        deposit(payment) {
+        async deposit(payment) {
```

Comment:
orthogonal to this PR: In `asyncFlow` discussion yesterday, somebody said the facade methods can't be async.